packages/catalog-process/src/services/validators.ts

```diff
@@ -59,15 +62,37 @@ export function assertHasNoDraftDescriptor(eservice: EService): void {
   }
 }
 
-export function validateRiskAnalysisOrThrow(
+export function buildRiskAnalysisValidationResultOrThrow(
```

Comment:
I had the [same impression](https://github.com/pagopa/interop-be-monorepo/pull/263#discussion_r1522978159) about this name 😅 (although I see your point)
Also, if we remove the `schemaOnly` parameter it's probably clearer to specify the functionality in the name.
IMO previous naming was clear enough, so my suggestion would be `validateRiskAnalysisSchemaOrThrow`

Reply:
✔️  makes sense, fixing this and merging right away